crawler/cv/cv/spiders/a36kr.py

```diff
@@ -15,25 +17,32 @@ def parse(self, response):
         # filename = 'data/'+response.url.split("/")[-1]
         # with open(filename, 'wb') as f:
         #     f.write(response.body)
-        obj = response.css('.js-react-on-rails-component')\
+        obj = response.css('.js-react-on-rails-component') \
             .xpath('@data-props').extract()
         result = json.loads(obj[0])
         post = result['data']['post']
 
-        nowdate = datetime.datetime.now()
+        nowdate = datetime.datetime.utcnow()
```

Comment:
Because differing time zones can cause conflicts， should be `now` instead of `utcnow`.You know what you should do?